Makefile

```diff
@@ -5,15 +5,15 @@ RELEASE_DIR = target/release
 MANPAGE = $(ASSETS_DIR)/alacritty.man
 TERMINFO = $(ASSETS_DIR)/alacritty.info
 COMPLETIONS_DIR = $(ASSETS_DIR)/completions
-COMPLETIONS = $(COMPLETIONS_DIR)/_alacritty \
-			  $(COMPLETIONS_DIR)/alacritty.bash \
-			  $(COMPLETIONS_DIR)/alacritty.fish
+COMPLETIONS =   $(COMPLETIONS_DIR)/_alacritty \
+		$(COMPLETIONS_DIR)/alacritty.bash \
+		$(COMPLETIONS_DIR)/alacritty.fish
```

Comment:
Generally I'd be fine with aligning these tbh. EIther way is fine to me so this doesn't really need to be changed if you ask me.